Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
```

Comment:
If we're going to include `Pipfile` and `Pipfile.lock` (which I think is a good idea) they should probably include all the dependencies, including those in `docs/requirements.txt`?

Reply:
Sure that can be added to the `[dev-packages]` section.

Reply:
Done with f210f28edf7d4d980117f42988227d1390f8aa32